### PR TITLE
Add swapfile support

### LINKED ISSUE #4: add swap space
digitaloean doesn't have swap space by default, enable it.

### PR DESCRIPTION
**Pull request by binary1230**
fix #4 

note: most of this diff is just adding the module, there's only 3 real lines of changes on our local deploy

### CODE REVIEW

File: puppet/manifests/roles.pp [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 ) {
   include ssh
 
+  if (!$::is_vagrant) {
+    include swap
+  }
+
   include '::ntp'
 
   class { 'timezone':
```
Comment (binary1230) [inline puppet/manifests/roles.pp]: actual changes part 1 of 2

File: puppet/manifests/swap.pp [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class swap {
+  # see https://github.com/petems/petems-swap_file
+  swap_file::files { 'default':
+    ensure   => present,
+  }
+}
```
Comment (binary1230) [inline puppet/manifests/swap.pp]: actual changes part 2 of 2

**Comment by binary1230** [comment]
btw, we shouldn't have to copy module code like this, instead should be able to use puppet-librarian. but since we're potentially radically altering the interaction of puppet + docker in the near future, just doing it the old lazy way which involves copying the module code.

**Comment by kitsuta** [comment]
+1 OMHBITY

**Comment by binary1230** [comment]
I'll add this in after Magstock, I realized I may have to check for the existince of $::is_vagrant before using it (that variable won't exist on non-vagrant installs).  Puppet might be smart enough to handle this automatically, but I hadn't actually tested it on a staging server yet.

**Comment by binary1230** [comment]
this should be OK now, I added getvar() to check for existince of ::is_vagrant

**Pull request merged**